test/e2e/cli/mongocli/mongocli.go

```diff
@@ -95,5 +95,5 @@ func GetClusterStateName(projectID string, clusterName string) string {
 
 func GetVersionOutput() {
 	session := cli.Execute("mongocli", "--version")
-	session.Wait()
+	session.Wait(10)
```

Comment:
default wait() timeout for this suite is equal to 1min. Why?

Reply:
discussed p2p: this is just to avoid waiting for 1 min as this command is expected to return fast